debian/changelog

```diff
@@ -1,3 +1,9 @@
+unattended-upgrades (1.0) UNRELEASED; urgency=medium
```

Comment:
hm, lets call it 0.100 ?

Reply:
@mvo5 is there anything else you would like to see in a release to call it 1.0?

Reply:
@rbalint I was thinking about this a bit and I think I agree - its pretty feature complete (once we deal with kernels better) and I don't think there is any feature missing at this point. So :+1: for 1.0. What are you plans with the kernel? Will that happen before 18.04?

Reply:
@mvo5 If nothing urgent comes up, before weekend. ;-)

Reply:
@rbalint yay!